### PR TITLE
CVMFS Cloud

### PR DESCRIPTION
**Pull request by almahmoud**
Needs small extra change in `_helpers.tpl` with https://github.com/galaxyproject/galaxy-helm/pull/271

Friends with https://github.com/CloudVE/galaxy-cvmfs-csi-helm/pull/11

### CODE REVIEW

File: galaxy/values.yaml [hunk 1]
```diff
@@ -166,7 +166,7 @@ ingress:
   hosts:
     - host: ~
       paths:
-        - path: "/"
```
Comment (afgane) [inline galaxy/values.yaml:169]: Why this change?

Comment (almahmoud) [reply galaxy/values.yaml:169]: Because on line 165 above it's always been set to default to `/galaxy` and the two paths should not mismatch.

**Review by afgane — APPROVED** [review]
